Classes/Mautic/OAuth.php

```diff
@@ -27,10 +28,22 @@ class OAuth implements AuthInterface
      */
     protected $baseUrl;
 
-    public function __construct(AuthInterface $authorization, string $baseUrl)
+    /**
+     * @var string
+     */
+    protected $accesToken;
```

Comment:
You can avoid the `@var` annotations here be setting `protected $accessToken = ''`